addons/mail/static/src/model/model_field.js

```diff
@@ -419,16 +430,23 @@ class ModelField {
      *
      * @private
      * @param {mail.model|mail.model[]} newValue
+     * @param {Object} [param1={}]
+     * @param {boolean} [param1.hasToVerify=true] whether the value has to be
+     *  verified @see `_verifyRelationalValue`
      * @returns {mail.model[]}
      */
-    _setRelationConvertX2ManyValue(newValue) {
+    _setRelationConvertX2ManyValue(newValue, { hasToVerify = true } = {}) {
```

Comment:
name of function is misleading. It doesn't set but convert (convert should be the prefix).

Reply:
The original idea was "convert in the context of set relation" to keep all the related methods with the same prefix.

Reply:
Misleading name is a bigger problem than the one of proximity of related functions. Each time this function is used, I'm wondering what it does... or more precisely why it "sets", just to see that it's not a set but a conversion (+ check).